src/Sets/Zonotope.jl

```diff
@@ -533,3 +508,35 @@ function _vertices_list_iterative(c::AbstractVector{N}, G::AbstractMatrix{N}; ap
 
     return apply_convex_hull ? convex_hull!(vlist) : vlist
 end
+
+@inline function _angles(point::AbstractVector{N})  where{N<:Real}
+    (atan(point[2], point[1]) /π * 180 + 360)%360
+end
+
+function _single_quadrant_vertices_enum(G::AbstractMatrix{N}, sorted::Bool=true) where{N<:Real}
+    if !sorted
+        G = sortslices(G, dims=2, by=_angles)
+     end
+    return VPolygon(2*cumsum(hcat(G, -G), dims=2) .- sum(G, dims=2))
+end
+
+function _vertices_list_2D(c::AbstractVector{N}, G::AbstractMatrix{N}, apply_convex_hull::Bool) where {N<:Real}
```

Comment:
```suggestion
function _vertices_list_2D(c::AbstractVector{N}, G::AbstractMatrix{N}; apply_convex_hull::Bool) where {N<:Real}
```
This fixes the build error.